### PR TITLE
docs: Add EMR docs

### PR DESCRIPTION
**Pull request by anusudarsan**

### CODE REVIEW

**Comment by anusudarsan** [comment]
@petroav @amrutagokhale can you take a look please?

File: presto-docs/src/main/sphinx/aws/emr.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+=============
+Presto on EMR
+=============
+
+TODO: Add an intro and links to appropriate presto-admin docs
```
Comment (anusudarsan) [inline presto-docs/src/main/sphinx/aws/emr.rst]: I dint know if you were already adding anything here , @petroav . The EMR resizing should probably be a new section here.

Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: Yes. I have all my docs written so I'll send it all to you and you can add it.

**Comment by anusudarsan** [comment]
Updated PR with Anton's work on "Resizing cluster" doc

File: presto-docs/src/main/sphinx/aws/emr.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+    sudo /path/to/presto-admin -i /path/to/your.pem configuration deploy
+    sudo /path/to/presto-admin -i /path/to/your.pem server restart
+
+6. Terminate the node using the AWS console or the AWS CLI tool.
+
+Troubleshooting
+===============
+
+* If you are using the RPM for installing Presto, make sure you use the RPM built for RPM Package Manager version > 4.6. If you are using an RPM which is not compatible with an EMR node, you may get the following error during installation:
```
Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: In my section of the docs I manually wrap lines to 80 characters except for long commands that can't be broken. Can you do that in this part as well?

Comment (anusudarsan) [inline presto-docs/src/main/sphinx/aws/emr.rst]: yes. will do

Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: Where can a user find the correctly build RPM?

I would add a sentence along the lines of "Such an RPM should be one of the artifacts you downloaded".

File: presto-docs/src/main/sphinx/aws/emr.rst [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+built for RPM Package Manager version > 4.6. If you are using an RPM which 
+is not compatible with an EMR node, you may get the following error during 
+installation:
+
+.. code-block:: none
+
+    headerRead failed: hdr load: BAD
+   
+* EMR deploys ``hive 1.0.0`` version which has a bug with 
+``ALTER TABLE..RENAME TO`` command. Thus while using Presto's Hive Connector
```
Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: -> you get -> you may run into the following

I would also remove "thus"

File: presto-docs/src/main/sphinx/aws/emr.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+``ALTER TABLE..RENAME TO`` command. Thus while using Presto's Hive Connector 
+you get:
+
+.. code-block:: none
+
+    presto:default> ALTER TABLE test_table RENAME TO new_test_table;
+    
+    Query 20160322_155117_00002_ai777 failed: InvalidOperationException(message:Unable to access old location hdfs://node:8020/user/hive/warehouse/test_table for table default.test_table)
+    
+You can workaround this issue, by changing the ownership of
```
Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: remove the comma

Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: -> changing the ownership of _the_ `/user/hive/warehouse` directory

File: presto-docs/src/main/sphinx/aws/emr.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+you get:
+
+.. code-block:: none
+
+    presto:default> ALTER TABLE test_table RENAME TO new_test_table;
+    
+    Query 20160322_155117_00002_ai777 failed: InvalidOperationException(message:Unable to access old location hdfs://node:8020/user/hive/warehouse/test_table for table default.test_table)
+    
+You can workaround this issue, by changing the ownership of 
+``/user/hive/warehouse`` directory to:
```
Comment (petroav) [inline presto-docs/src/main/sphinx/aws/emr.rst]: -> directory to hive:hadoop as shown below:

**Comment by petroav** [comment]
Looks good. Make sure it compiles before pushing.

File: presto-docs/src/main/sphinx/aws/emr.rst [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+installation:
+
+.. code-block:: none
+
+    headerRead failed: hdr load: BAD
+   
+* EMR deploys ``hive 1.0.0`` version which has a bug with 
+``ALTER TABLE..RENAME TO`` command. Thus while using Presto's Hive Connector 
+you get:
+
```
Comment (amrutagokhale) [inline presto-docs/src/main/sphinx/aws/emr.rst]: I think the following text will be a bit more clarifying:

EMR deploys `hive 1.0.0` version which has a bug with `ALTER TABLE..RENAME TO` command. When running this command in Presto using the Hive connector, you may run into the following:

**Comment by amrutagokhale** [comment]
Looks good % comment.

**Pull request merged**